src/pages/wallet/WalletStatementPage.tsx

```diff
@@ -70,7 +70,7 @@ function WalletStatementPage({walletStatement, route}: WalletStatementPageProps)
     const year = yearMonth?.substring(0, 4) || getYear(new Date());
     const month = yearMonth?.substring(4) || getMonth(new Date());
     const monthName = format(new Date(Number(year), Number(month) - 1), CONST.DATE.MONTH_FORMAT);
-    const title = translate('statementPage.title', year, monthName);
+    const title = translate('statementPage.title', {year, monthName});
```

Comment:
If we have updated the params style here, please update in the en.ts and es.ts too. Otherwise, the output string will be this `undefined [object Object] statement`

https://github.com/Expensify/App/blob/3ba42e684f75f143d6e7a85e62d23b7d561d7155/src/languages/en.ts#L2352-L2353

```diff
- title: (year, monthName) => `${monthName} ${year} statement`,
+ title: ({year, monthName}) => `${monthName} ${year} statement`,
```